setup.cfg

```
@@ -1,22 +1,40 @@
 [metadata]
 name = equistore
-description = "A specialized data storage format suited to atomistic machine learning needs and more."
-long_description = file: README.rst
+url = https://github.com/lab-cosmo/equistore
+description = A specialized data storage format suited to atomistic machine learning needs and more.
+long_description = file: README.rst, LICENSE.rst
```

```suggestion
long_description = file: README.rst, LICENSE.rst
```

I've never seen the license included in the long description. There is no way to manually include the license file in a separate field?

---

Me neither. But I took a look at the setuptools docs and they suggest this.

https://setuptools.pypa.io/en/latest/userguide/declarative_config.html

I think we can go with the `license_files` attribute though.